assets/js/admin/admin-scripts.js

```diff
@@ -974,8 +978,12 @@ var give_setting_edit = false;
 						});
 						self.process_step(parseInt(response.step), data, self);
 					}
-					// Reset the form for preventing multiple ajax request.
-					$('#give-tools-recount-form')[0].reset();
+
+					if ( true === reset_form ) {
```

Comment:
@DevinWalker `Give_Reports` and `Give_Exports` share same `form` tag and `process_step` fn, so to reset form @raftaar1191 edit `process_step` which is running for both. It will not create any issue but it is better to refactor code that allows us to maintain both.  I can see we are using process step many places in admin backend it will be a good idea if we will create a global function within `Give` object as we are doing in the frontend. We can do that after `2.0`.

@DevinWalker Let me know what do you think